projects/packages/jetpack-mu-wpcom/src/features/help-center/class-help-center.php

```diff
@@ -214,13 +214,15 @@ function ( $wp_admin_bar ) {
 			);
 		}
 
-		wp_enqueue_script(
-			'wpcom-upsell-support-link-handler',
-			plugins_url( 'build/wpcom-upsell-support-link-handler/wpcom-upsell-support-link-handler.js', Jetpack_Mu_Wpcom::BASE_FILE ),
-			array(),
-			filemtime( Jetpack_Mu_Wpcom::BASE_DIR . 'build/wpcom-upsell-support-link-handler/wpcom-upsell-support-link-handler.js' ),
-			true
-		);
+		if ( $variant === 'wp-admin' || $variant === 'wp-admin-disconnected' ) {
```

Comment:
Cannot we use `is_admin` instead?

Anyway, I think we can remove this script and revert https://github.com/Automattic/jetpack/pull/44199 altogether, if you add the `data-target="wpcom-help-center"` attribute to the support link in wpcom (see https://github.com/Automattic/wp-calypso/pull/104172).